### PR TITLE
Changes for release2.3.0

### PR DESCRIPTION
**Pull request by skrakau**
- updated Pandas version from 1.1.5 to 1.4.3 for processes depending only on Pandas
- Removed the `BUSCO_PLOT` process. The [new BUSCO version 5.4.3](https://gitlab.com/ezlab/busco/-/commit/f4744bf0479c562a8a9a6630f31584a286f75b39) would have required additional renaming of the `short_summary` files, since they are called "short_summary.domain.\*" or "short_summary.specific_lineage.\*" within this pipeline (see [here](https://github.com/nf-core/mag/blob/5329440f663c54ce28c2362a8ba167ccb9b60e14/modules/local/busco.nf#L15)). 
  However, for the results when run with auto lineage selection, the current processing was inaccurate. One would need to select somehow, if the "short_summary.domain.\*" or "short_summary.specific_lineage.\*" file is used for this plot, while not both are present for all bins and using the most specific summary might not be best for comparison. Since anyway these plots are probably not designed for metagenomic bin comparisons, we decided to drop this process.

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
  - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/mag/tree/master/.github/CONTRIBUTING.md)
  - [ ] If necessary, also make a PR on the nf-core/mag _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: subworkflows/local/busco_qc.nf [hunk 1]
```diff
@@ -5,13 +5,11 @@
 params.busco_db_options            = [:]
 params.busco_options               = [:]
 params.busco_save_download_options = [:]
-params.busco_plot_options          = [:]
 params.busco_summary_options       = [:]
 
 include { BUSCO_DB_PREPARATION            } from '../../modules/local/busco_db_preparation'        addParams( options: params.busco_db_options            )
```
Comment (d4straub) [inline subworkflows/local/busco_qc.nf:10]: Uh, there is still `addParams`, shouldnt that be deleted by now? Those are anyway not used, correct? This is now all in `modules.config`.

File: subworkflows/local/busco_qc.nf [hunk 2]
```diff
@@ -5,13 +5,11 @@
 params.busco_db_options            = [:]
 params.busco_options               = [:]
 params.busco_save_download_options = [:]
-params.busco_plot_options          = [:]
 params.busco_summary_options       = [:]
```
Comment (d4straub) [inline subworkflows/local/busco_qc.nf:8]: Those also shouldnt be needed any more...

File: CHANGELOG.md [hunk 3]
```diff
@@ -21,6 +21,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - [#340](https://github.com/nf-core/mag/pull/340),[#368](https://github.com/nf-core/mag/pull/368),[#373](https://github.com/nf-core/mag/pull/373) - Update to nf-core 2.7.2 `TEMPLATE` (by @jfy133, @d4straub, @skrakau)
 - [#373](https://github.com/nf-core/mag/pull/373) - Removed parameter `--enable_conda`. Updated local modules to new conda syntax and updated nf-core modules (by @skrakau)
 - [#385](https://github.com/nf-core/mag/pull/385) - CAT also now runs on unbinned contigs as well as binned contigs (added by @jfy133)
+- [#399](https://github.com/nf-core/mag/pull/399/files) - Removed BUSCO_PLOT process, which generated undocumented BUSCO plots (`*.busco_figure.png`) (by @skrakau).
```
Comment (jfy133) [inline CHANGELOG.md:24]:
```suggestion
- [#399](https://github.com/nf-core/mag/pull/399/files) - Removed undocumented BUSCO_PLOT process (previously generated `*.busco_figure.png` plots unsuitable for metagenomics) (by @skrakau).
```

Or something like that, I suggested putting undocumented earlier to emphasise it was never _really_ supported as it wasn't a documented output ;)

**Review by jfy133 — APPROVED** [review]

File: subworkflows/local/busco_qc.nf (hunk 1 above)
Comment (skrakau) [reply subworkflows/local/busco_qc.nf:10]: Can I just delete this or do I need to add something somewhere?

Comment (d4straub) [reply subworkflows/local/busco_qc.nf:10]: _theoretically_ they shouldn't be used and can be deleted.

Comment (d4straub) [reply subworkflows/local/busco_qc.nf]: addParams appears also in gtdbtk.nf & binning.nf & binning_preparation.nf ...

**Pull request merged**